pylzy/lzy/api/v1/remote/runtime.py

```diff
@@ -353,13 +355,25 @@ def __build_graph(
             else:
                 docker_image = None
 
-            conda_yaml: Optional[str]
-            if call.env.conda_yaml_path:
-                with open(call.env.conda_yaml_path, "r") as file:
-                    conda_yaml = file.read()
+            python_env: Optional[Operation.PythonEnvSpec]
+
+            if docker_image and call.env.docker_only:
+                python_env = None  # execute in bash env if without conda
```

Comment:
let's drop comment here or replace it with `don't use conda for 'docker_only' ops`